driver_cpl/cime_config/buildexe

```diff
@@ -20,10 +20,10 @@ logger = logging.getLogger(__name__)
 def _main_func():
 ###############################################################################
 
-    caseroot, libroot, _ = parse_input(sys.argv)
+    caseroot, bldroot, libroot = parse_input(sys.argv)
 
     logger.info("Building a single executable version of target coupled model")
-
+    os.chdir(bldroot)
```

Comment:
Is this chdir call (as well as the one on line 51) necessary? At a glance, it looks like some of the other builds are managed without a chdir, though maybe I'm missing something. I just ask because I think we're trying to reduce the use of chdirs (#462). If it's really necessary, then I'm fine with it.